String Calculator/js/stringHandler.js

```diff
@@ -0,0 +1,28 @@
+class StringCalculator{
+
+    operations = {
+        "+": (a, b) => a + b,
+        "-": (a, b) => a - b,
+        "*": (a, b) => a * b,
+        "/": (a, b) => a / b
+    }
+    
+    calculate(str) {
+    
+        let part = str.split(' ');
+
+        let result = +part[0];
+        if(part.length % 2 != 0){
+
+            for(let i = 1; i < part.length-1; i += 2){
```

Comment:
what if there is an expression like `1 + 1 / 2`? I'm getting `1` but should `1.5`. The implementation does not check for the operator priority.